### PR TITLE
CI init

### PR DESCRIPTION
**Pull request by jah3**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <version>3.2.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
+    <artifactId>demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>demo</name>
+    <description>Demo project for Spring Boot</description>
+    <properties>
+        <java.version>17</java.version>
```
Comment (taragrammatonn) [inline pom.xml:17]: Update to Java 21

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.2.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
+    <artifactId>demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>demo</name>
+    <description>Demo project for Spring Boot</description>
```
Comment (taragrammatonn) [inline pom.xml:15]: Please add corresponding description

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.2.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
+    <artifactId>demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>demo</name>
```
Comment (taragrammatonn) [inline pom.xml:14]: Name fix please

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.2.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
+    <artifactId>demo</artifactId>
```
Comment (taragrammatonn) [inline pom.xml:12]: Fix artifactId like sample-bank

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.2.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
```
Comment (taragrammatonn) [inline pom.xml:11]: Replace with md.maib

**Review by taragrammatonn — CHANGES_REQUESTED** [review]

**Comment by taragrammatonn** [comment]
squash pls

File: pom.xml [hunk 6]
```diff
@@ -8,15 +8,26 @@
...
-    <name>demo</name>
-    <description>Demo project for Spring Boot</description>
+    <name>bank-sample</name>
+    <description>demo bank-sample</description>
     <properties>
-        <java.version>17</java.version>
+        <java.version>21</java.version>
     </properties>
 
+    <dependencies>
```
Comment (taragrammatonn) [inline pom.xml:20]: unneeded changes at this step

**Comment by Sp1ithy** [comment]
Please **squash** commits and use **force push**
It gonna be 1 logical commit including all fixes

**Review by taragrammatonn — APPROVED** [review]

**Pull request merged**